examples/rgbpp/shared/utils.ts

```diff
@@ -0,0 +1,35 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+/**
+ * Save ckbVirtualTxResult to a log file
+ * @param ckbVirtualTxResult - The ckbVirtualTxResult to save
+ * @param exampleName - Example name used to distinguish different log files
+ */
+export const saveCkbVirtualTxResult = (ckbVirtualTxResult: unknown, exampleName: string) => {
+  try {
+    // Define log file path
+    const logDir = path.resolve(__dirname, '../logs');
+    const logFilePath = path.join(logDir, `${exampleName}-ckbVirtualTxResult.log`);
```

Comment:
### Suggestion
I think a `timestmap` prefix or suffix to the log file name is helpful to distinguish the log files created at different times.

```suggestion
    const timestamp = new Date().toISOString().replace(/:/g, '-'); // Replace colons with hyphens
    const logFilePath = path.join(logDir, `${exampleName}-${timestamp}-ckbVirtualTxResult.log`);
```